content/doc/addons/mongodb/_index.md

```diff
@@ -78,4 +78,20 @@ Note that these features are available for all our databases add-ons, in additio
 
 **The managed databases provide an advantage if you want to avoid the complexity of installing and deploying databases** and if you want simple-to-use management and monitoring tools.
 
-{{% content/managed-services %}}
+## 🔑 Rights and permissions
```

Comment:
As this part is almost the same for different databases, maybe we should: 
- Keep a shortcode integration
- Use general language across databases 
- Explain what's specific to an add-on in a dedicated part

Reply:
I agree with @aurrelhebert since even in the similar text parts there are subtle differences between each database (like the specified db role / permissions)